data/lm/README.rst

```diff
@@ -7,22 +7,24 @@ You can download the LibriSpeech corpus with the following commands:
 
     wget http://www.openslr.org/resources/11/librispeech-lm-norm.txt.gz
 
+|
 
-Then use the `generate_lm.py` script to generate `lm.binary` and `vocab-500000.txt`.
+| Then use the ``generate_lm.py`` script to generate ``lm.binary`` and ``vocab-500000.txt``.
+| As input you can use a plain text (e.g. ``file.txt``) or gzipped (e.g. ``file.txt.gz``) text file with one sentence in each line.
 
-As input you can use a plain text (e.g. `file.txt`) or gzipped (e.g. `file.txt.gz`) text file with one sentence in each line.
-
-If you are not using the DeepSpeech docker container, you have to build `KenLM <https://github.com/kpu/kenlm>`_ first and then pass the build directory to the script.
+If you are using the DeepSpeech docker container, you can use ``--kenlm_bins /DeepSpeech/native_client/kenlm/build/bin/``.
```

Comment:
Fix that wording at the same time. There is no such thing as "The DeepSpeech docker container" that we can seriously consider usable and tested. The `Dockerfile` we have is only built and it's provided as a best-effort basis.